themes/prism-synthwave84.css

```diff
@@ -0,0 +1,142 @@
+/*
+ * Synthwave '84 Theme originally by Robb Owen [@Robb0wen] for Visual Studio Code
+ * Demo: https://marc.dev/demo/prism-synthwave84
+ *
+ * Ported for PrismJS by Marc Backes [@themarcba]
+ */
+
+ code[class*="language-"],
+ pre[class*="language-"] {
+     color: #f92aad;
+     text-shadow: 0 0 2px #100c0f, 0 0 5px #dc078e33, 0 0 10px #fff3;
+     background: none;
+     font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
+     font-size: 1em;
+     text-align: left;
+     white-space: pre;
+     word-spacing: normal;
+     word-break: normal;
+     word-wrap: normal;
+     line-height: 1.5;
+ 
+     -moz-tab-size: 4;
+     -o-tab-size: 4;
+     tab-size: 4;
+ 
+     -webkit-hyphens: none;
+     -moz-hyphens: none;
+     -ms-hyphens: none;
+     hyphens: none;
+ 
+ }
+ 
+ /* Code blocks */
+ pre[class*="language-"] {
+     padding: 1em;
+     margin: .5em 0;
+     overflow: auto;
+ }
+ 
+ :not(pre)>code[class*="language-"],
+ pre[class*="language-"] {
+     background-color: transparent !important;
+     background-image: linear-gradient(to bottom, #2a2139 75%, #34294f);
+ }
+ 
+ /* Inline code */
+ :not(pre)>code[class*="language-"] {
+     padding: .1em;
+     border-radius: .3em;
+     white-space: normal;
+ }
+ 
+ .token.comment,
+ .token.block-comment,
+ .token.prolog,
+ .token.doctype,
+ .token.cdata {
+     color: #8e8e8e;
+ }
+ 
+ .token.punctuation {
+     color: #ccc;
+ }
+ 
+ .token.tag,
+ .token.attr-name,
+ .token.namespace,
+ .token.number,
+ .token.unit,
+ .token.hexcode,
+ .token.deleted {
+     color: #e2777a;
+ }
+ 
+ .token.property,
+ .token.selector {
+     color: #72f1b8;
+     text-shadow: 0 0 2px #100c0f, 0 0 10px #257c5575, 0 0 35px #21272475;
+ }
+ 
+ .token.function-name {
+     color: #6196cc;
+ }
+ 
+ .token.boolean,
+ .token.selector .token.id,
+ .token.function {
+     color: #fdfdfd;
+     text-shadow: 0 0 2px #001716, 0 0 3px #03edf975, 0 0 5px #03edf975, 0 0 8px #03edf975;
+ 
+ }
+ 
+ .token.class-name {
+     color: #fff5f6;
+     text-shadow: 0 0 2px #000, 0 0 10px #fc1f2c75, 0 0 5px #fc1f2c75, 0 0 25px #fc1f2c75;
+ }
+ 
+ .token.constant,
+ .token.symbol {
+     color: #f92aad;
+     text-shadow: 0 0 2px #100c0f, 0 0 5px #dc078e33, 0 0 10px #fff3;
+ }
+ 
+ .token.important,
+ .token.atrule,
+ .token.keyword,
+ .token.selector .token.class,
+ .token.builtin {
+     color: #f4eee4;
+     text-shadow: 0 0 2px #393a33, 0 0 8px #f39f0575, 0 0 2px #f39f0575;
+ }
+ 
+ .token.string,
+ .token.char,
+ .token.attr-value,
+ .token.regex,
+ .token.variable {
+     color: #f87c32;
+ }
+ 
+ .token.operator,
+ .token.entity,
+ .token.url {
+     color: #67cdcc;
+ }
+ 
+ .token.important,
+ .token.bold {
+     font-weight: bold;
+ }
+ 
+ .token.italic {
+     font-style: italic;
+ }
+ 
+ .token.entity {
+     cursor: help;
+ }
+ 
+ .token.inserted {
+     color: green;
+ }
```

Comment:
Needs a newline @ end of file.

Reply:
Thanks for reviewing the pull request. And this is terribly embarrassing, but I've never created a pull request before, therefore I don't know how this all works...

You pointed out some flaws in the contribution. Do I need to change those, or did you already do the changes?

Reply:
Yeah, make the suggested changes, add them as commits to the branch, and push that branch to your remote. Then the PR will be updated with them.

Reply:
@themarcba This still hasn't been fixed, has it?

Reply:
@RunDevelopment it is.